src/main/java/jaiden/ui/Parser.java

```diff
@@ -0,0 +1,45 @@
+package jaiden.ui;
+
+import jaiden.command.AddCommand;
+import jaiden.command.Command;
+import jaiden.command.DeleteCommand;
+import jaiden.command.ExitCommand;
+import jaiden.command.FindCommand;
+import jaiden.command.ListCommand;
+import jaiden.command.MarkCommand;
+import jaiden.command.UnknownCommand;
+
+/**
+ * Class to parse user input into executable command.
+ */
+public class Parser {
+    /**
+     * Parses user input into executable command.
+     *
+     * @param command User input.
+     * @return Corresponding command.
+     */
+    public Command parse(String command) {
+        String[] commands = command.split(" ");
+        switch (commands[0]) {
+        case "list":
+        case "show":
+            return new ListCommand(commands);
+        case "mark":
+        case "unmark":
+            return new MarkCommand(commands);
+        case "todo":
+        case "deadline":
+        case "event":
+            return new AddCommand(commands);
+        case "delete":
+            return new DeleteCommand(commands);
+        case "bye":
+            return new ExitCommand(commands);
+        case "find":
+            return new FindCommand(commands);
+        default:
+            return new UnknownCommand(commands);
```

Comment:
The way you handle the cases is very clean!
```suggestion
        case "list":
        case "show":
            return new ListCommand(commands);
        case "mark":
        case "unmark":
            return new MarkCommand(commands);
        case "todo":
        case "deadline":
        case "event":
            return new AddCommand(commands);
        case "delete":
            return new DeleteCommand(commands);
        case "bye":
            return new ExitCommand(commands);
        case "find":
            return new FindCommand(commands);
        default:
            return new UnknownCommand(commands);
```